src/api-client.ts

```diff
@@ -17,6 +19,9 @@ export class RevAiApiClient {
     accessToken: string;
     version: string;
     instance: AxiosInstance;
+    private TEXT_OUTPUT = 'text/plain';
+    private JSON_OUTPUT = 'application/vnd.rev.transcript.v1.0+json';
+    private SRT_OUTPUT = 'application/x-subrip';
```

Comment:
1. minor -  are these considered content-types? Should the name reflect this?

2. These should either be const at file scope of static instance members.

Reply:
Actually, these should be string enums https://www.typescriptlang.org/docs/handbook/enums.html
```
enum ContentType {
Text = 'text/plain',
Json = 'application/vnd.rev.transcript.v1.0+json',
Srt = 'application/x-subrip'
}
```